scalameter-core/src/main/scala/org/scalameter/PicklerBasedKey.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter
+
+import org.scalameter.picklers.Pickler
+
+
+/* A mixin for keys whose values require pickler based serialization. */
+trait PicklerBasedKey[T] {
```

Comment:
Should this be `Serializable`? I see that it is used for `Parameter` below, but both are not `Serializable`.